pkg/operator/operator.go

```diff
@@ -39,6 +41,11 @@ func New(metricsAddr string) (*Operator, error) {
 		return nil, fmt.Errorf("unable to register the thanos querier controller with the manager: %w", err)
 	}
 
+	ok := func(_ *http.Request) error { return nil }
```

Comment:
Do we want to do anything extra with the request here?

Reply:
I think it's ok. We could even use https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.3/pkg/healthz#Ping

Reply:
Updated.